src/content/upgrades/merge/index.md

```diff
@@ -1,113 +1,98 @@
 ---
 title: The Merge
-description: Learn about The Merge - when Mainnet Ethereum joins the Beacon Chain coordinated proof-of-stake system.
+description: Learn about The Merge - when Mainnet Ethereum adopted proof-of-stake.
 lang: en
 template: upgrade
 sidebar: true
 image: ../../../assets/upgrades/merge.png
-summaryPoint1: Soon, the current Ethereum Mainnet will merge with the Beacon Chain proof-of-stake system.
-summaryPoint2: This will mark the end of proof-of-work for Ethereum, and the full transition to proof-of-stake.
-summaryPoint3: This sets the stage for future scaling upgrades including sharding.
-summaryPoint4: The Merge will reduce Ethereum's energy consumption by ~99.95%.
+summaryPoint1: Ethereum Mainnet uses proof-of-stake, but this wasn't always the case.
+summaryPoint2: The transition from the original proof-of-work mechanism to proof-of-stake was called The Merge.
```

Comment:
```suggestion
summaryPoint2: The upgrade from the original proof-of-work mechanism to proof-of-stake was called The Merge.
```
Not a strong preference here but "upgrade" might be more explicit on how the transition happened (or else explicitly mention a hard fork?).